src/App.js

```diff
@@ -1,85 +1,38 @@
 import './App.css';
-import { ImCart, ImUser, ImPlus, ImMinus } from 'react-icons/im';
+import { useEffect, useState } from 'react';
+import Header from './components/Header';
+import Body from './components/Body';
+import SideMenu from './components/SideMenu';
 
 function App() {
-  return (
-    <div className="wrapper">
-      <header>
-        <div className="headerLeft">
-          <img width={60} height={50} src="/img/logo.png" />
-          <div className="headerInfo">
-            <h3> Sneacker shop</h3>
-            <p>Магазин </p>
-          </div>
-        </div>
-        <ul className="headerRight">
-          <li>
-            <ImCart className="cart" />
-            <span>100 RUB</span>
-          </li>
-          <li>
-            <ImUser className="profileLogo" />
-          </li>
-        </ul>
-      </header>
-      <h1>Все кроссовки</h1>
-      <div className="content">
-        <div className="cards">
-          <img width={200} height={200} src="/img/sneakers1.jpeg" />
-          <p>Кроссовки мужские NIKE</p>
-          <div className="cardBottom">
-            <div className="priceButton">
-              <span> Цена:</span>
-              <b> 12 999 RUB</b>
-            </div>
-            <button>
-              <ImPlus className="button" />
-            </button>
-          </div>
-        </div>
+  const [cardsInfo, setCardsInfo] = useState([]);
 
-        <div className="cards">
-          <img width={200} height={200} src="/img/sneakers1.jpeg" />
-          <p>Кроссовки мужские NIKE</p>
-          <div className="cardBottom">
-            <div className="priceButton">
-              <span> Цена:</span>
-              <b> 12 999 RUB</b>
-            </div>
-            <button>
-              <ImPlus className="button" />
-            </button>
-          </div>
-        </div>
+  useEffect(() => {
+    (async function getProductData() {
+      const response = await fetch('https://fakestoreapi.com/products');
```

Comment:
try {
  const response = await fetch('https://website');
} catch (error) {
  // TypeError: Failed to fetch
  console.log('There was an error', error);
}

Reply:
fixed